_posts/2020-08-28-acess.md-the-most-open-privacy-first-emr-solution.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Access.MD – the most open, privacy first, EMR solution"
+date: "2020-08-28 20:18:45 +0300"
+author: "Ameer Alqam"
+---
+
+## Who we are?
+
+OpenMyMed is not-for-profit foundation focused on fostering Access.MD, an open source EMR and telemedicine platform, as well as many upcoming OSS projects relating to the medical industry.
```

Comment:
We cant say not-for-profit or foundation since we arent formally registered

Reply:
```suggestion
OpenMyMed is a foundation focused on fostering Access.MD, an open source EMR and telemedicine platform, as well as many upcoming OSS projects relating to the medical industry.
```